ports/arrayfire/portfile.cmake

```
@@ -91,7 +92,7 @@ vcpkg_cmake_install()
 
 vcpkg_copy_pdbs()
 
-if(VCPKG_TARGET_IS_OSX)
+if(VCPKG_TARGET_IS_OSX OR VCPKG_TARGET_IS_LINUX)
```

```suggestion
if(NOT VCPKG_TARGET_IS_WINDOWS)
```
or simpy control `AF_INSTALL_CMAKE_DIR`.
https://github.com/arrayfire/arrayfire/blob/f4157374e73a2140293e68de7a00174fa675da6a/CMakeModules/AFInstallDirs.cmake#L52-L59

---

Done